includes/class-newspack-newsletters.php

```diff
@@ -282,6 +321,77 @@ public static function rest_api_init() {
 				],
 			]
 		);
+		\register_rest_route(
+			'newspack-newsletters/v1/',
+			'typography/(?P<id>[\a-z]+)',
+			[
+				'methods'             => \WP_REST_Server::EDITABLE,
+				'callback'            => [ __CLASS__, 'api_set_typography' ],
+				'permission_callback' => [ __CLASS__, 'api_administration_permissions_check' ],
+				'args'                => [
+					'id'    => [
+						'validate_callback' => [ __CLASS__, 'validate_newsletter_id' ],
+						'sanitize_callback' => 'absint',
+					],
+					'key'   => [
+						'validate_callback' => [ __CLASS__, 'validate_newsletter_typography_key' ],
+						'sanitize_callback' => 'sanitize_text_field',
+					],
+					'value' => [
+						'validate_callback' => [ __CLASS__, 'validate_newsletter_typography_value' ],
+						'sanitize_callback' => 'sanitize_text_field',
+					],
+				],
+			]
+		);
+	}
+
+	/**
+	 * Set typography meta.
+	 *
```

Comment:
I think a comment should be added here describing the need for a custom endpoint to set meta, because it might not be obvious.

Reply:
Good idea. https://github.com/Automattic/newspack-newsletters/pull/210/commits/121ca5c15dd34231c57abe892fdd52a72a6b09bb